std_msgs/Header header
float32 current_charge
float32 discharge_rate
bool mode
bool fault
uint8 fault_code